infra/vsts_agent_ubuntu_20_04_startup.sh

```diff
@@ -72,9 +72,10 @@ apt-get install -qy \
     xdg-utils \
     wget
 
+## Currently broken, temporarily disabling as this prevents machine startup
 # Taken from https://cloud.google.com/logging/docs/agent/logging/installation
```

Comment:
The installation steps in this referenced document have changed and thus steps below are no longer correct

Reply:
My guess is that you only needed to remove the apt-key add line and leave agent install. This also depends on whether Google added logging agent by default into Ubuntu build.